packages/playground/src/pages/DropinUMD/manual.js

```diff
@@ -63,9 +63,9 @@ export async function initManual() {
         onOrderRequest: async (resolve, reject) => {
             resolve(await createOrder({ amount }));
         },
-        onOrderCancel: async order => {
+        onOrderCancel: async (order, actions) => {
             await cancelOrder(order);
-            checkout.update({ paymentMethodsResponse: await getPaymentMethods({ amount, shopperLocale }), order: null, amount });
+            actions.resolve({ amount });
```

Comment:
What happens if the `actions.reject('Something went wrong') ` gets called? Do we display this error in the UI?

Reply:
No error is going to be displayed in the UI. There is no UI design for it, if it's well implemented it shouldn't fail besides a failed network request. So we just throw network error now.